flink/core/src/test/java/cz/o2/proxima/flink/core/BatchLogSourceFunctionTest.java

```diff
@@ -332,7 +332,12 @@ public void collectWithTimestamp(StreamElement element, long timestamp) {
 
           @Override
           public void go() throws Exception {
-            sourceFunction.run(context);
+            try {
+              sourceFunction.run(context);
+            } catch (Exception e) {
+              e.printStackTrace();
```

Comment:
debug?

Reply:
yes, I'll try to unify the test cases as well, I think it should be pretty straightforward now